packages/simcore-sdk/tests/unit/test_node_ports_common_file_io_utils.py

```diff
@@ -1,31 +1,82 @@
 # pylint: disable=redefined-outer-name
 # pylint: disable=unused-argument
 
-from asyncio import BaseEventLoop
+from typing import AsyncIterable, Iterable
 
 import pytest
-from aiohttp import web
-from aiohttp.test_utils import TestClient
+from aiohttp import ClientResponse, ClientSession
+from aioresponses import aioresponses
 from simcore_sdk.node_ports_common.file_io_utils import (
     ExtendedClientResponseError,
+    _check_for_aws_http_errors,
     _raise_for_status,
 )
 
-TEST_ERROR = "OPSIE there was an error here"
+A_TEST_ROUTE = "http://127.0.0.1:1249/test-route"
 
 
 @pytest.fixture
-def test_client(loop: BaseEventLoop, aiohttp_client) -> TestClient:
-    async def raise_error(request):
-        return web.Response(body=TEST_ERROR, status=web.HTTPBadRequest.status_code)
+def mock_service() -> Iterable[aioresponses]:
+    with aioresponses() as mock:
```

Comment:
We already have a fixture that does this in ``pytest_simcore``:

https://github.com/ITISFoundation/osparc-simcore/blob/3f88ad3d6340b40d616db8c4d0eee80d47da44ed/packages/pytest-simcore/src/pytest_simcore/aioresponses_mocker.py#L17

Reply:
I seem to have missed it, even after looking at it :\